src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocParagraphCheck.java

```diff
@@ -140,7 +140,7 @@ else if (ast.getType() == JavadocTokenTypes.HTML_ELEMENT
      */
     private void checkEmptyLine(DetailNode newline) {
         final DetailNode nearestToken = getNearestNode(newline);
-        if (!isLastEmptyLine(newline) && nearestToken.getType() == JavadocTokenTypes.TEXT
+        if (nearestToken.getType() == JavadocTokenTypes.TEXT
```

Comment:
I couldn't find a way to kill this. I believe this is because the check is trying to find a newline all alone, and ending new lines are placed as a child of the last tag.